public/app/features/alerting/unified/components/rule-editor/AlertRuleForm.tsx

```diff
@@ -130,7 +131,7 @@ export const AlertRuleForm = ({ existing, prefill }: Props) => {
   const type = watch('type');
   const dataSourceName = watch('dataSourceName');
 
-  const showStep2 = Boolean(type && (type === RuleFormType.grafana || !!dataSourceName));
+  const showDataSourceDependantStep = Boolean(type && (type === RuleFormType.grafana || !!dataSourceName));
```

Comment:
I'm not sure about why this check is necessary 🤔

Reply:
Do we really need to check if `RuleFormType.grafana` ?